aries-site/src/examples/components/textarea/TextAreaValidationExample.js

```diff
@@ -27,6 +27,7 @@ export const TextAreaValidationExample = () => {
             }
             setValue(event.target.value);
           }}
+          resize="vertical"
```

Comment:
I don't know if we should/need to add this to this this validation example, or should we be removing it from the /TextAreaExample.js ?

I do think we should add it to any examples where the FormField is contained in a Box with a width set.

Reply:
Okay yes that is fair, we don't necessarily need to restrict horizontal resizing for these examples. I will remove it from "TextAreaExample" to be consistent.